index.html

```diff
@@ -12,6 +12,10 @@
     <script src="./common.js" class="remove"></script>
     <script class="remove"
       src="https://cdn.jsdelivr.net/gh/digitalbazaar/respec-vc@2.0.0/dist/main.js"></script>
```

Comment:
That's the code that provides the multiple tabs for DI/VC-JWT examples in the spec... without it, we'd have to manually update all of the "secured" examples in the spec.

Reply:
ah nice, so where do I open the PR to make the examples for JWT "v2" ?

Reply:
PRs welcome here: https://github.com/digitalbazaar/respec-vc/pulls

I'm going to try to update it to use the new v2 Data Integrity stuff before CR. It would be nice to have the new VC-JWT stuff supported then as well.